submissions/morenko/memory-game/js/index.js

```diff
@@ -0,0 +1,182 @@
+// создадим массив с картами
```

Comment:
1. **Only** english in code :)
2. Redundant comments - don’t use stuff ‘———————————‘ or “here I will sort my array”. The first one is ugly, second - redundant because the reviewer will understand that you do sort, from your code. The only situation in which you should use comments is when you need to describe why something is going on and this is 1% of all situations :)